pkg/registry/apis/secret/encryption/migrator/reencrypt.go

```diff
@@ -37,7 +37,7 @@ func (s simpleSecret) ReEncrypt(ctx context.Context, namespace string, secretsSr
 				return err
 			}
 
-			encrypted, err := secretsSrv.Encrypt(ctx, namespace, decrypted, encryption.WithoutScope())
+			encrypted, err := secretsSrv.Encrypt(ctx, namespace, decrypted, contracts.EncryptWithoutScope())
```

Comment:
A note but I wonder if we need this extra param, shouldn't we always encrypt WithScope, and that scope being the namespace? Gotta ask Michael about it